assets/xml/objects/object_skb.xml

```diff
@@ -1,64 +1,65 @@
 ﻿<Root>
+    <!-- Assets for Stalchild -->
     <File Name="object_skb" Segment="6">
-        <Animation Name="object_skb_Anim_0009E4" Offset="0x9E4" />
-        <Animation Name="object_skb_Anim_0015EC" Offset="0x15EC" />
-        <Animation Name="object_skb_Anim_001D1C" Offset="0x1D1C" />
-        <Animation Name="object_skb_Anim_002190" Offset="0x2190" />
+        <Animation Name="gStalchildIdleAnim" Offset="0x9E4" />
+        <Animation Name="gStalchildSaluteAnim" Offset="0x15EC" />
+        <Animation Name="gStalchildSwingOnBranch" Offset="0x1D1C" />
```

Comment:
```suggestion
        <Animation Name="gStalchildSwingOnBranchAnim" Offset="0x1D1C" />
```
Sorry, missed this one

Reply:
Sorry, lots of silly mistakes in this one...

Reply:
No worries, it's tough to get all the little details correct.

Reason why the build failed is because you have to update this name where it's used in code

Reply:
Updated all instances of SwingOnBranch to SwingOnBranchAnim, i think the build passes now.